Jenkinsfile

```diff
@@ -3,7 +3,7 @@ node {
     def kibanaVersion = '7.7.1'
     def scmVars = checkout scm
     sh "env"
-    def imageName = "test-image:${env.BUILD_ID}"
+    def imageName = "${env.JOB_BASE_NAME}-test-image:${env.BUILD_ID}"
```

Comment:
env.BRANCH_NAME refers to the branch it is currently building. It is more explanatory and definitive than the JOB_BASE_NAME, which can give different values based on the setting of the Jenkins project. Therefore, I would recommend to try with BRANCH_NAME. Same comment goes to other branches.

Reply:
Cool, updated PR to have env.BRANCH_NAME